src/update_assisted_service_cm.py

```diff
@@ -40,6 +40,7 @@
     ("HW_VALIDATOR_MIN_CPU_CORES_SNO", ""),
     ("HW_VALIDATOR_MIN_RAM_GIB_SNO", ""),
     ("ENABLE_UPGRADE_AGENT", "true"),
+    ("JWKS_URL", ""),
```

Comment:
This should be the default right ?
https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/certs

Reply:
It should be but i didn't want to change current configurations, only to add the option to insert it as env param, should change it to default?

Reply:
yes looks like it doesn't work